src/replace_scp.py

```diff
@@ -0,0 +1,95 @@
+"""Replace SCP.
+
+Purpose:
+    Replace DETACH_SCP_ID SCP with ATTACH_SCP_ID SCP defined in the associated env vars
+Environment Variables:
+    LOG_LEVEL: (optional): sets the level for function logging
+            valid input: critical, error, warning, info (default), debug
+    DETACH_SCP_ID: id of the policy to detach
+    ATTACH_SCP_ID: id of the new policy to attach
+    ASSUME_ROLE_NAME: Name of role to assume
+"""
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+import collections
+import logging
+import os
+import sys
+
+import boto3
+
+# Standard logging config
+DEFAULT_LOG_LEVEL = logging.INFO
+LOG_LEVELS = collections.defaultdict(
+    lambda: DEFAULT_LOG_LEVEL,
+    {
+        "CRITICAL": logging.CRITICAL,
+        "ERROR": logging.ERROR,
+        "WARNING": logging.WARNING,
+        "INFO": logging.INFO,
+        "DEBUG": logging.DEBUG,
+    },
+)
+
+ATTACH_SCP_ID = os.environ["ATTACH_SCP_ID"]
+DETACH_SCP_ID = os.environ["DETACH_SCP_ID"]
+
+# Lambda initializes a root logger that needs to be removed in order to set a
+# different logging config
+root = logging.getLogger()
+if root.handlers:
+    for handler in root.handlers:
+        root.removeHandler(handler)
+
+logging.basicConfig(
+    format="%(asctime)s.%(msecs)03dZ [%(name)s][%(levelname)-5s]: %(message)s",
+    datefmt="%Y-%m-%dT%H:%M:%S",
+    level=LOG_LEVELS[os.environ.get("LOG_LEVEL", "").lower()],
+)
+
+log = logging.getLogger(__name__)
+
+# Get client
+org_client = boto3.client("organizations")
+
+
+def lambda_handler(event, context):  # pylint: disable=unused-argument
+    """Replace scp policy lambda handler."""
+    log.debug("AWS Event:%s", event)
```

Comment:
Using the lambda powertools, this can be removed, replaced by a decorator. See: https://github.com/plus3it/terraform-aws-org-new-account-support-case/blob/86f5a29ae46241971db88ebff55fa7cb1d001f6f/lambda/src/new_account_support_case.py#L139